crates/turbopack-ecmascript/src/minify.rs

```diff
@@ -63,6 +64,8 @@ pub async fn minify(path: Vc<FileSystemPath>, code: Vc<Code>) -> Result<Vc<Code>
             let top_level_mark = Mark::new();
 
             Ok(compiler.run_transform(handler, false, || {
+                let program = program.fold_with(&mut paren_remover(Some(&comments)));
```

Comment:
Can the minifier check if that fixer was applied?

Considering "Fixes ast nodes before printing so semantics are preserved.", it feels like   somewhere we should crash if the fixer isn't applied. 

In other words: When would you ever want to not preserve semantics?

Reply:
It should be done by `codegen`, but we can't know if it's applied from codegen